### ISSUE TITLE
Scrollbar slider is opaque, hides scrollbar decorations/indicators

### ISSUE THREAD

**Issue by driftwoodstudio** [nilshartmann/vscode-blue-light-theme#1]
Using: Blue Light Theme 2022

Scrollbar slider is opaque white, which causes it to cover/hide the colored indicators put into the scrollbar (e.g. red marker to indicate problem in code at that portion of file). 

Solved this by setting a permanent override in local user settings (choosing values for myself that work ok with both light and dark theme choices I use):

```
    "workbench.colorCustomizations": {
        "scrollbarSlider.background": "#0000ff23",
        "scrollbarSlider.activeBackground": "#ff000023",
        "scrollbarSlider.hoverBackground": "#00ff0023",    
    }
```
Note use of '23' as alpha value, so the slider has some transparency. 

Small thing, but makes it frustrating when trying to "scroll to part of the code that has an error".

Love the theme otherwise. Thank you for making and distributing it!

**Comment by nilshartmann** [comment]
Hi @driftwoodstudio ,

glad that you like the theme 😊.

Thanks a lot for your feedback, I never recognized the missing transparency, as I normally don't look at the scrollbar indicators. Anyway: I will apply your suggestions in a new version (propably need some days).

**Comment by nilshartmann** [comment]
Hi @driftwoodstudio,

thanks again for your feedback.

I published version 0.6.4 that now uses transparent background color for the slides. I use different colors than yours to match it better with the other scheme colors.

**Comment by driftwoodstudio** [comment]
Great, I’ll grab the new release. 😀

-- Bill
